Exoft.Gamification.Api.Services.Interfaces/Services/IAchievementService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Exoft.Gamification.Api.Common.Models;
+using Exoft.Gamification.Api.Data.Core.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exoft.Gamification.Api.Services.Interfaces.Services
+{
+    public interface IAchievementService : IService<Achievement>
+    {
+        Task<Achievement> GetAchievementAsync(Guid Id);
```

Comment:
Once again... service should return Model, not Entity

Reply:
Remove this service... Implementation is incorrect. It was not  required in scope of  Authentication/authorization task....